src/const/external.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Event type object with event name strings.
+ *
+ * @ko 이벤트 이름 문자열들을 담은 객체
+ * @type {object}
+ * @property {"holdStart"} HOLD_START - holdStart event<ko>holdStart 이벤트</ko>
+ * @property {"holdEnd"} HOLD_END - holdEnd event<ko>holdEnd 이벤트</ko>
+ * @property {"moveStart"} MOVE_START - moveStart event<ko>moveStart 이벤트</ko>
+ * @property {"move"} MOVE - move event<ko>move 이벤트</ko>
+ * @property {"moveEnd"} MOVE_END - moveEnd event<ko>moveEnd 이벤트</ko>
+ * @property {"change"} CHANGE - change event<ko>change 이벤트</ko>
+ * @property {"restore"} RESTORE - restore event<ko>restore 이벤트</ko>
+ * @property {"select"} SELECT - select event<ko>select 이벤트</ko>
+ * @property {"needPanel"} NEED_PANEL - needPanel event<ko>needPanel 이벤트</ko>
+ * @example
+ * import { EVENTS } from "@egjs/flicking";
+ * EVENTS.MOVE_START; // "MOVE_START"
+ */
+export const EVENTS: {
+  READY: "ready";
+  RESIZE: "resize";
+  HOLD_START: "holdStart";
+  HOLD_END: "holdEnd";
+  MOVE_START: "moveStart";
+  MOVE: "move";
+  MOVE_END: "moveEnd";
+  CHANGE: "change";
+  RESTORE: "restore";
+  SELECT: "select";
+  NEED_PANEL: "needPanel";
+  VISIBLE_CHANGE: "visibleChange";
+  CONTENT_ERROR: "contentError";
+} = {
+  READY: "ready",
+  RESIZE: "resize",
+  HOLD_START: "holdStart",
+  HOLD_END: "holdEnd",
+  MOVE_START: "moveStart",
+  MOVE: "move",
+  MOVE_END: "moveEnd",
+  CHANGE: "change",
+  RESTORE: "restore",
+  SELECT: "select",
+  NEED_PANEL: "needPanel",
+  VISIBLE_CHANGE: "visibleChange",
+  CONTENT_ERROR: "contentError"
+};
+
+export const ALIGN: {
+  PREV: "prev";
+  CENTER: "center";
+  NEXT: "next";
+} = {
+  PREV: "prev",
+  CENTER: "center",
+  NEXT: "next"
+};
+
+export const DIRECTION: {
+  PREV: "PREV";
+  NEXT: "NEXT";
+} = {
+  PREV: "PREV",
+  NEXT: "NEXT"
+};
```

Comment:
Add `as const` to the constant. There is no need to set the type separately.

Reply:
This really helps, thanks